### ISSUE TITLE
global "core.history": ["Cmd+R"],

### ISSUE THREAD

**Issue by monouser7dig** [marta-file-manager/marta-issues#173]
```
"core.history": ["Cmd+R"],
```
is currently tap specific, I'd like it to be global unless that seems like a bad workflow/ use to you/ nobody else uses it like that

**Comment by warpkanal** [comment]
Yes, would prefer global as well (rendering #222 not needed, unless a global history would be configurable)
